### ISSUE TITLE
2016-01-26: Update after a break

### ISSUE THREAD

**Issue by andkov** [IALSA/Portland-physical-cogntive#4]
## News and Updates
1. See new section on Projects in the [main README](https://github.com/IALSA/IALSA-2015-Portland#projects).  I've edited https://github.com/IALSA/IALSA-2015-Portland/blob/master/projects/README.md to reflect the current state of our thinking about the projects stemming from the Portland conference. In my mind, each track develops into a repo that develop and host the publication. 
2. Workflow of renaming and reclassifying submitted models is described in the new [./manipulation/README.md](https://github.com/IALSA/Portland-physical-cognitive/blob/master/manipulation/README.md). 
3. The new report gives the current [cognitive domain map[(https://github.com/IALSA/Portland-physical-cognitive/blob/master/analysis/model_space/Cog-Domain-Map.md), reflecting the map provided in the .csv files (see manipulation/README)
4. New [stencil for IALSA-2015-Portland](https://github.com/IALSA/ialsa-2015-portland-stencil) which pulls all the lessons we've learned (and learning) from Portland. One of its goals is to conduct the Portland conference with perfect hindsight. [ialsa-2016-victoria](https://github.com/IALSA/ialsa-2016-victoria) is being developed to do just that: to start with the raw data sets for individual studies, and then go through the motions of coordinated analysis from data sets to dynamic reports.  Assuming we would like to get better at these live conferences we need a rehearsal. 
5. The new cognitive domain map is ready
   ![cog-domain-map](https://raw.githubusercontent.com/IALSA/Portland-physical-cognitive/master/analysis/model_space/cog_domain_map/domain_map-1.png)

`perception` and `verbal reasoning` need to be placed within the existing order and assigned colors to reflect the underlying continuum

**Comment by andkov** [comment]
@ampiccinin , please sort the following domains in the order in which you would like them to be sorted on the graph. Currently this order is: 

```
knowledge 
language  
fluency
memory
workmemory
executive f
visuospatial
speed
mental status

perception
verbal reasoning
```

**Comment by ampiccinin** [comment]
1. Order OK.
   Visuospatial and verbal reasoning should both appear under domain "Fluid reasoning"
   Is perception the line orientation task? let's just put it under mental status.

**Comment by ampiccinin** [comment]
We should also edit the naming convention as I see several errors/inconsistencies (https://github.com/IALSA/IALSA-2015-Portland/blob/master/libs/images/model_naming_convention.png):

As far as I know, "muscle" should just be "grip". What is "hand" and how does it differ from "Grip"? Is this just a labelling issue?

Chair, flamingo and TUG are all just alternate lower body strength measures that supplement or approximate "walking". There should be fewer "domains" and these variations should show up under "measure".

Walking should not appear under cognitive

Hope this helps.

**Comment by andkov** [comment]
@ampiccinin , great, thanks, I'll walk through your comments point by point when updating the graph and will get back to you.

**Comment by andkov** [comment]
@ampiccinin , yes, thank you for pointing out the inconsistencies in the naming convention. it was created in the early stages of the project and does not reflect our current thinking. For example, we don't use construct names in the name of the model any more. I need to think how to update this convention to be in accord with what's coming up next.

**Comment by andkov** [comment]
@ampiccinin , just updated the naming convention 

![model_naming_convention](https://cloud.githubusercontent.com/assets/2608073/13400125/34c2d1e2-debb-11e5-8ec1-11ea2a3b9d68.png)

However, note the further mechanisms of model submission will downplay the importance of the filenames. However, @wibeasley , such filenames, I would like to argue, should remain in our practice. It provides those manholes to the individual model we would like to keep at all times.

**Comment by wibeasley** [comment]
> @wibeasley , such filenames, I would like to argue, should remain in our practice. It provides those manholes to the individual model we would like to keep at all times.

The lookup tables should facilitate manhole covers too.  But I'm willing to try your way.

With the naming, can you make two changes that will allow us to more easily validate the names:
1. dashes instead of underscores.
2. all lower case.  (I'm ok with "nophys" instead of "noPhys" or "no-phys".  This makes it similar to your "digitsymbol" and "proserecall".)

**Comment by andkov** [comment]
I agree with the proposed changed and have altered the convention accordingly. 
![model_naming_convention](https://cloud.githubusercontent.com/assets/2608073/13401598/98a59fae-dec1-11e5-909e-ccaccbb7be9f.png)

As for the file names, manholes is one thing. Almost even more important is the convenience it offers me to think about the whole process. I think the fact that the name of the file is descriptive to humans offers me a sense of control. So i'm not rooting for the file names per se, but rather for some objects for the mind to hold on to during the description of the work flow. I'd be happy to abandon it after the launch of the look up tables; right now my imagination fails me.

**Comment by andkov** [comment]
@ampiccinin , items previously classified in `visuospacial` and `verbal
reasoning` are now classified into the domain `fluid reasoning`. Below is the latest version of the domain map graph. The colors are still experimental and I haven't implemented the suggestions by @wibeasley yet. I would like to have a separate session on prettifying it . 
![domain_map-1](https://cloud.githubusercontent.com/assets/2608073/13402129/c2cbee3a-dec3-11e5-9e75-7658a324808d.png)
